test/integration/policy-imagemanifest_test.go

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package integration
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	policiesv1 "github.com/stolostron/governance-policy-propagator/api/v1"
+	"github.com/stolostron/governance-policy-propagator/test/utils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8stypes "k8s.io/apimachinery/pkg/types"
+
+	"github.com/stolostron/governance-policy-framework/test/common"
+)
+
+const policyIMVURL = "https://raw.githubusercontent.com/stolostron/policy-collection/main/stable/SI-System-and-Information-Integrity/policy-imagemanifestvuln.yaml"
+const policyIMVName = "policy-imagemanifestvuln"
+const subName = "container-security-operator"
+
+var ImageManifestLabel = Label("policy-collection", "stable", "SVT")
+
+var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the "+policyIMVName+" policy", ImageManifestLabel, func() {
+	It("stable/"+policyIMVName+" should be created on the Hub", func() {
+		By("Creating the policy on the Hub")
+		_, err := utils.KubectlWithOutput(
+			"apply", "-f", policyIMVURL, "-n", userNamespace, "--kubeconfig="+kubeconfigHub,
+		)
+		Expect(err).To(BeNil())
+
+		By("Patching the namespaceSelector to use the " + userNamespace + " namespace")
```

Comment:
Don't think this `namespaceSelector` patch makes sense. The `imagemanifestvulns` CRs should exist as long as pod in the namespace is using a vulnerable.